### PR TITLE
Fix doc for missing values.

### LINKED ISSUE #1944: building doc is failing for the release 0.10.1
I found the following page fails
http://xarray.pydata.org/en/stable/examples/weather-data.html

### PR DESCRIPTION
**Pull request by fujiisoup**
- [x] Closes #1944

### CODE REVIEW

File: doc/api.rst [hunk 1]
```diff
@@ -113,6 +113,22 @@ Indexing
...
+   Dataset.isnull
+   Dataset.notnull
+   Dataset.count
+   Dataset.dropna
+   Dataset.fillna
+   Dataset.ffill
+   Dataset.bfill
+   Dataset.interpolate_na
+   Dataset.where
+
```
Comment (fujiisoup) [inline doc/api.rst]: I think it would be nice if we could see docstrings for these methods in rtd.

Comment (shoyer) [reply doc/api.rst]: Works for me, but these should be removed from `api-hidden.rst` now.

**Comment by fujiisoup** [comment]
I'm wondering this doc is some commits ahead of the stable (e.g. it has docs about `rolling.construct`).
Should I modify stable branch?

File: doc/api.rst (hunk 1 above)
Comment (fujiisoup) [reply doc/api.rst]: Done.

**Comment by shoyer** [comment]
@fujiisoup What I do for updating the stable docs is to cherry-pick commits with doc fixes to the stable branch. As long as those apply cleanly (which is usually the case) we can update the stable docs.

**Comment by shoyer** [comment]
I just cherry-picked the merge commit from this PR into `stable`. So RTD should automatically update when the build finishes in ~15 minutes.

**Comment by fujiisoup** [comment]
Thanks, @shoyer 

But this part 
http://xarray.pydata.org/en/stable/computation.html#missing-values
is still failing (I think this was working on my environment).
Is there any missing dependencies on rtd?

**Pull request merged**